connector/src/test/scala/com/basho/riak/spark/rdd/connector/RiakConnectorTests.scala

```diff
@@ -17,12 +31,14 @@ class RiakConnectorTests {
 
   @Test
   def openSessionWithDefaultParametersShouldReturnSessionFromPool(): Unit = {
-    val session_1 = connector_1.openSession()
+    mockSessionCache(2, new RiakClient(mock(classOf[RiakCluster])))
```

Comment:
I've considered such approach, but unfortunately this is not an option. Because "mockSessionCache" method receives size of cache as an argument and this parameter could be varying for different tests based on test logic, so I decided not to use it in @Before method